include/swift/ABI/MetadataValues.h

```diff
@@ -106,6 +106,47 @@ inline ClassFlags &operator|=(ClassFlags &a, ClassFlags b) {
   return a = (a | b);
 }
 
+/// Flags that go in a MethodDescriptor structure.
+class MethodDescriptorFlags {
+public:
+  typedef uint32_t int_type;
+  enum class Kind {
+    InstanceMethod,
```

Comment:
I feel like static vs instance is orthogonal to method vs accessor, given that class property accessors can appear in the vtable as well.

Reply:
Ah, yeah, that's true.  Maybe 'static' makes more sense as a separate bit.

Reply:
I was definitely just not thinking about static properties.

Reply:
Maybe it could be a field, like static vs. mutating vs. non-mutating vs. consuming?

Reply:
On the other hand, maybe that belongs more in a type description.

Reply:
Submitted #11712 to address this.